### PR TITLE
fix: Substitute negative and invalid values (e.g. `"fish"` in `headingoffset="fish"`) with the default value (`0`)

### LINKED ISSUE #13: Non-long/negatives via JS?
given...
```html
<div id="a" headingoffset="-2">
   <h1>A</h1>
</div>

<div id="b" headingoffset="fish">
   <h1>B</h1>
</div>
```

While the invalid values (https://html.spec.whatwg.org/#limited-to-only-non-negative-numbers) don't actively seem to screw up the `aria-level` that is applied (or isn't), the property seems to give other values:

```
> console.log(a.headingOffset) 
-2

> console.log(b.headingOffset)
NaN
```

Is that right? It's not how some similar things like `colSpan` work...  Perhaps what should happen here is that there should be a conceptual default value of 0 (that's what `colSpan` does) which should be returned by that API if it can't parse.  I imagine this is what you are doing in practice for computing anyway, shouldn't it log the same?

### PR DESCRIPTION
**Pull request by smockle**
Fixes #13

### CODE REVIEW

File: headingoffset-polyfill.test.html [hunk 1]
```diff
@@ -40,6 +40,14 @@
           it("default value when unset", () => {
             expect(container.headingOffset).to.equal(0);
           });
+          it("default value when invalid", () => {
+            container.headingOffset = "fish";
+            expect(container.headingOffset).to.equal(0);
+          });
+          it("allows negative values", () => {
+            container.headingOffset = -1;
+            expect(container.headingOffset).to.equal(-1);
```
Comment (bkardell) [inline headingoffset-polyfill.test.html:49]: I thought it does not allow negative values, and this should also return 0?

Comment (smockle) [reply headingoffset-polyfill.test.html:49]: You are correct! Here are the relevant lines from the implementation of `GetHeadingOffset` in [ax_node_object.cc (line 2968–2994)](https://chromium-review.googlesource.com/c/chromium/src/+/5445406/13/third_party/blink/renderer/modules/accessibility/ax_node_object.cc):

```C
if (ancestorHeadingOffset > 0) {
  offset += ancestorHeadingOffset;
}
```

In other words, if `ancestorHeadingOffset` is negative, it’s _not_ added into `offset`.

Comment (smockle) [reply headingoffset-polyfill.test.html:49]: (Converting this PR to draft while I fix that.)

Comment (smockle) [reply headingoffset-polyfill.test.html]: Fixed via https://github.com/smockle/headingoffset-polyfill/pull/14/commits/36d82071c9dc6c363adcad4a21c185462a2a80c4!

**Review by bkardell — COMMENTED** [review]
lgtm

**Pull request merged**